backend/src/Service/CaptainFinancialSystem/CaptainFinancialSystemAccordingOnOrderService.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Service\CaptainFinancialSystem;
+
+use App\AutoMapping;
+use App\Entity\CaptainFinancialSystemAccordingOnOrderEntity;
+use App\Manager\CaptainFinancialSystem\CaptainFinancialSystemAccordingOnOrderManager;
+use App\Response\CaptainFinancialSystem\CaptainFinancialSystemAccordingOnOrderResponse;
+
+class CaptainFinancialSystemAccordingOnOrderService
+{
+    private CaptainFinancialSystemAccordingOnOrderManager $CaptainFinancialSystemAccordingOnOrderManager;
```

Comment:
Variable name must be small letter. And check all places in which it is being used.